src/makeRobot.js

```diff
@@ -38,7 +38,80 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const template = {
+    x: 0,
+    y: 0,
```

Comment:
```suggestion
    x: 0,
    y: 0,
```
Опис задачі каже
`- The coordinates of the robot must be stored in the object coords, the keys x and y inside the robot.`
Потрібно створити об'єкт coordinates, а x та y мають бути всередині цього об'єкта, а не прямо у робота